### DISCUSSION TITLE
Public Collègien·ne

### DISCUSSION THREAD

**Discussion by LHuberson** [unistra/esup-immersup-demandes#5] [3- Création de compte pour venir en immersion]
Bonjour, notre service est tête de cordée, a ce titre nous sommes amenés a organiser des actions à destination des collégien·ne·s. Nous aimerions pouvoir permettre des inscriptions de ce public. Les actions menées peuvent être soit sur le campus soit dans les établissements. La plupart du temps se sont les enseignant·e·s qui inscrivent un groupe d'élèves.

**Comment by LHuberson** [comment]
J'ai oublié de préciser les données pouvant être utiles : nom, prénom, date de naissance, mail, collège, niveau d'étude
Il pourrait être intéressant de préciser "Collège Encordé" ou "Élève encordé"

**Comment by efinkbeiner5** [comment]
A priori lié aux inscription en cohortes. A étudier.

**Reply by efinkbeiner5** [reply]
Tout est liée aux inscription en cohorte, pas d'immersion individuels pour les collégiens qui sont plutôt concernée par des visites

**Comment by efinkbeiner5** [comment]
Solution mise en place dans la version 4 de immersup. On peut déclarer des collèges (dans la même table de référence que les lycées) et une nouvelle case à cocher a été créé "Autoriser les immersions individuelles". Cette case doit être cochée pour tous les lycée qu veulent envoyer des lycéens en immersions individuelles. Elle ne le sera donc pas pour les collèges.

**Comment by efinkbeiner5** [comment]
Déployé en version 4 septembre 2024
